### PR TITLE
show target selection dialog when there is no config

### LINKED ISSUE #198: No target selection dialog when there is no config
### Bug Description

When there is no config at all, no target selection dialog starts.

### Steps to Reproduce

1. Delete any mirrord configuration file from the project.
2. Run with mirrord.
3. Don't get a target selection dialog.
4. Cry deeply.

### Additional Info

Target selection happens under the condition that the config path is not null, that's the problem.
https://github.com/metalbear-co/mirrord-intellij/blob/88039fe2bcf662a33c3e2b853ba7f46db1a96f27/modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordExecManager.kt#L134

No `configPath` -> no target selection.

### PR DESCRIPTION
**Pull request by t4lz**
fixed #198

### CODE REVIEW

File: modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordConfigAPI.kt [hunk 1]
```diff
@@ -30,6 +30,7 @@ class InvalidConfigException(path: String, reason: String) : MirrordError("faile
  * @throws InvalidConfigException if config cannot be found or parsed.
  */
 fun isTargetSet(config: String?): Boolean {
+    config ?: return false
```
Comment (t4lz) [inline modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordConfigAPI.kt:33]: Actually not really necessary for this fix, just seemed appropriate to me. Does it make sense or should I remove it?

File: modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordExecManager.kt [hunk 2]
```diff
@@ -114,7 +114,7 @@ class MirrordExecManager(private val service: MirrordProjectService) {
         // Find the mirrord config path, then call `mirrord verify-config {path}` so we can display warnings/errors
         // from the config without relying on mirrord-layer.
         val configPath = service.configApi.getConfigPath(mirrordConfigFromEnv)
-        MirrordLogger.logger.debug("MirrordExecManager.start: config path is $cli")
+        MirrordLogger.logger.debug("MirrordExecManager.start: config path is $configPath")
```
Comment (t4lz) [inline modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordExecManager.kt:117]: unrelated log fix

File: modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordExecManager.kt [hunk 3]
```diff
@@ -131,7 +131,9 @@ class MirrordExecManager(private val service: MirrordProjectService) {
 
         MirrordLogger.logger.debug("Verified Config: $verifiedConfig, Target selection.")
 
-        val target = if (configPath != null && !isTargetSet(verifiedConfig?.config)) {
+        val targetSet = verifiedConfig?.let{ isTargetSet(it.config) } ?: false
+        val target = if (!targetSet) {
+            // There is no config file or the config does not specify a target, so show dialog.
```
Comment (t4lz) [inline modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordExecManager.kt:136]: Please double check the logic here.

File: modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordConfigAPI.kt (hunk 1 above)
Comment (Razz4780) [reply modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordConfigAPI.kt:33]: That's cool :+1:

**Review by Razz4780 — APPROVED** [review]
LGTM

**Pull request merged**